pages/vi/profiles/Nida8930.md

```diff
@@ -0,0 +1,20 @@
+# Nicholas Davis
+![alt-text](https://img1.southernliving.timeinc.net/sites/default/files/styles/4_3_horizontal_-_1200x900/public/1542301326/GettyImages-1033240660.jpg?itok=6Sh4z7Za "Tulsa, Ok")
+
+About Me
+----
+Hello Everyone,
+
```

Comment:
Everything is good and meets the minimum requirements, but lets add in the timezone and system being used wherever you see fit:

```
Then, in the new Markdown file, include the following information:
   * your name, location/time zone, and OS (including version)
   * a short description of yourself (so your fellow virtual interns know who you are)
```

Reply:
Hey Caleb,
Unless something isn't loading correctly and I'm seeing something different my time zone and OS info is located at the bottom of the page under the general info section. Let me know if there is an issue that I'm not seeing. Thanks!

Reply:
Nope. You are absolutely correct. I'm not sure why it didn't register in my brain. My eyes just glazed right over that.
Sorry about that.

Reply:
No worries, it's early, glad everything looks good.